packages/expo-contacts/android/src/main/java/expo/modules/contacts/Contact.kt

```diff
@@ -285,15 +285,31 @@ class Contact(var contactId: String) {
           .build()
       )
     }
+
+    // Flush all data from linked db
+    ops.add(getFlushOperation(CommonDataKinds.Event.CONTENT_ITEM_TYPE, rawContactId!!))
+    ops.add(getFlushOperation(CommonDataKinds.Email.CONTENT_ITEM_TYPE, rawContactId!!))
+    ops.add(getFlushOperation(CommonDataKinds.Im.CONTENT_ITEM_TYPE, rawContactId!!))
+    ops.add(getFlushOperation(CommonDataKinds.Phone.CONTENT_ITEM_TYPE, rawContactId!!))
+    ops.add(getFlushOperation(CommonDataKinds.StructuredPostal.CONTENT_ITEM_TYPE, rawContactId!!))
+    ops.add(getFlushOperation(CommonDataKinds.Relation.CONTENT_ITEM_TYPE, rawContactId!!))
+    ops.add(getFlushOperation(CommonDataKinds.Website.CONTENT_ITEM_TYPE, rawContactId!!))
+    ops.add(getFlushOperation(CommonDataKinds.Nickname.CONTENT_ITEM_TYPE, rawContactId!!))
```

Comment:
```suggestion
rawContactId?.let { id ->
  listOf(
    CommonDataKinds.Event.CONTENT_ITEM_TYPE,
    CommonDataKinds.Email.CONTENT_ITEM_TYPE,
    CommonDataKinds.Im.CONTENT_ITEM_TYPE,
    CommonDataKinds.Phone.CONTENT_ITEM_TYPE,
    CommonDataKinds.StructuredPostal.CONTENT_ITEM_TYPE,
    CommonDataKinds.Relation.CONTENT_ITEM_TYPE,
    CommonDataKinds.Website.CONTENT_ITEM_TYPE,
    CommonDataKinds.Nickname.CONTENT_ITEM_TYPE,
  ).forEach {
    ops.add(getFlushOperation(it, id))
  }
}
```
Let's remove some of the repetitive code

Reply:
I did something a bit different,
- I use an array instead of a list (I think it is more efficient)
- I extract it to a member close to baseModels, as both list are very close together, I think it is more future proof to keep side by side.

Thank you for the fast review !